### ISSUE TITLE
Primitive session handling

### ISSUE THREAD

**Issue by EddieOfAlhana** [thomaswelton/laravel-facebook#4]
Thomaswelton\LaravelFacebook\Facebook extends Facebook\Facebook which uses php's default session handling. This class should be inhertied from Facebook\BaseFacebook and use laravel \Session:: facade...

**Comment by thomaswelton** [comment]
Thanks, pull request would be really welcomed.

**Comment by thomaswelton** [comment]
Hmm, looking at this again I can see that this could possibly cause some issue if this isn't fixed right?
Would this work well at all in you have a multi server setup?

**Comment by EddieOfAlhana** [comment]
Nah, on a multi server setup if PHP is configured to use file session handling you would need to login separately to every server. :) For me right now that was not the issue, as currently we have a single server setup. I just needed some mock mastery for unit testing my app, therefore I could not come around fixing this.
